Archives/VolFile.h

```diff
@@ -63,18 +64,19 @@ namespace Archives
 			int paddedStringTableLength;
 			int paddedIndexTableLength;
 
-			size_t fileCount()
+			size_t fileCount() const
 			{
 				return filesToPack.size();
 			}
 		};
 
 		int ReadTag(int offset, const char *tagText);
-		bool WriteTag(int length, const char *tagText);
-		bool CopyFileIntoVolume(HANDLE inputFile, int size);
+		void WriteTag(SeekableStreamWriter& volWriter, int length, const char *tagText);
+		void CopyFileIntoVolume(SeekableStreamWriter& volWriter, HANDLE inputFile, int size);
```

Comment:
These two methods might only need a StreamWriter, rather than a SeekableStreamWriter. They are both sequential in nature.

Reply:
Good point. I'll update this before merging.